src/NumericalAlgorithms/LinearOperators/Divergence.hpp

```diff
@@ -86,7 +86,7 @@ namespace Tags {
 ///
 /// This tag inherits from `db::add_prefix_tag<Tags::div, Tag>`.
 template <typename Tag, typename InverseJacobianTag>
-struct DivCompute : db::add_tag_prefix<div, Tag>, db::ComputeTag {
+struct DivVariablesCompute : db::add_tag_prefix<div, Tag>, db::ComputeTag {
```

Comment:
[not necessarily in the scope of this PR] Should there be a static_assert in this struct that the tensors in the variables are of rank `tnsr::I`?

Reply:
This raises an interesting point. `DivVariablesCompute` operators on `F^i_{\alpha\beta}` tags, so each tag in the variables only needs to have its first index be `^i` and then the divergence is taken of each component. Suggestions on how to make this clearer in the documentation?

Reply:
Ah, I didn't realize this, so I agree we could clarify the documentation. What do you think something like this (please edit as you see best, this is intended as a draft)?
```
\brief Compute the divergence of a Variables

Computes the divergence of the every Tensor in the Variables represented by
`Tag`. The first index of each Tensor must be an upper spatial index, i.e.,
the first index must have type
`TensorIndexType<SomeDim, UpLo::Up, Frame::SomeFrame, IndexType::Spatial>`.
The divergence is computed in the frame `SomeFrame`, and `InverseJacobianTag`
must be associated with a map from `Frame::Logical` to `Frame::SomeFrame`.

Note that each tensor may have additional tensor indices — in this case the
divergence is computed for each additional index. For instance, a tensor
\f$F^i_{\alpha\beta}\f$ has divergence
\f$Div_{\alpha\beta} = \partial_i F^i_{\alpha\beta}\f$.
```

(I think the bit about `IndexType::Spatial` is correct? But I'm not totally sure)